src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -0,0 +1,388 @@
+'''
+Contains helpers that are used to get statistics (specifically number of
+regular files and directories and total size of data present under a given
+directory) and pass, print, log and convert them to human readable format
+conveniently.
+'''
+from os.path import join as os_path_join
+from logging import getLogger
+from uuid import uuid4
+
+from .operations.volume import open_volume_lockless, get_all_volnames
+from .operations.subvolume import open_clone_sv_pair_in_vol
+from .operations.clone_index import open_clone_index
+from .operations.resolver import resolve_grp_and_sv_names
+
+from mgr_util import RTimer
+from cephfs import ObjectNotFound
+
+log = getLogger(__name__)
+
+
+def get_size_h(num):
```

Comment:
I'm sure I commented this before - have you checked if mgr_utils::format_units() would suffice to be used rather than wiring these helpers up?

Reply:
I had checked it and used another but related function that was even better. I'll find out that patch and push it too.

Reply:
Done - https://github.com/ceph/ceph/compare/98b9949d8b5adcc5372f0129bf833606a2834964..28ab9ceaad17354830311cd474235b54c20c029d